frontend/src/styles/antd-adjustments.scss

```diff
@@ -25,6 +25,15 @@
   }
 }
 
+/*
+ * Forms
+ */
+
+.ant-form-item-label {
+  font-weight: bold;
```

Comment:
The change is effective for all the form labels on the app?

Reply:
Yes